omod-1.9/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs1_9/PersonController1_9Test.java

```diff
@@ -398,4 +398,14 @@ public void shouldCreateAPersonWithBooleanAttributeWithQuotes() throws Exception
 		assertEquals(givenName, person.getGivenName());
 		assertEquals("true", person.getAttribute(attributeId).getValue());
 	}
+	@Test
+	public void shouldEditAPersonBirthTime() throws Exception {
+		Person person = service.getPersonByUuid(getUuid());
+		String json = "{\"birthtime\": \"1970-01-01T20:20:00.000\"}";
+		SimpleObject response = deserialize(handle(newPostRequest(getURI() + "/" + getUuid(), json)));
+		assertNotNull(response);
+		Object responsePersonContents = PropertyUtils.getProperty(response, "person");
+		assertNotNull(responsePersonContents);
+		//assertEquals("1970-01-01T20:20:00.000+0000", PropertyUtils.getProperty(responsePersonContents, "birthtime").toString());
```

Comment:
Are all check failed after overriding getUpdatableProperties?

Reply:
Which checks?

Reply:
You are getting that error because you added your test to a class which runs from openmrs platform version `1.9.x`. You would need to add it to a class which runs in any openmrs platform version from 1.10.3 and above

Reply:
Yeah, I run test on other class above 1.9.x it’s not giving error.

Reply:
Therefore you need to move the test from that class.

Reply:
yeah, i move that test to PersonController2_2Test.java class, and now it's working. do ineed to commit tat changes?

Reply:
i committed those changes and resolved the merge conflict please look into it. if every things good, please merge it with master.